lib/sdf/link.rb

```diff
@@ -18,5 +18,39 @@ def each_sensor
         def pose
             Conversions.pose_to_eigen(xml.elements["pose"])
         end
+
+        # The link's inertial.
+        #
+        # @return Inertial struct
+        Inertial = Struct.new(:pose, :mass, :inertia)
+        def inertial
+            mass = read_float_child_element(xml, "mass", 1)
+            ixx = read_float_child_element(xml, "ixx", 1)
+            iyy = read_float_child_element(xml, "iyy", 1)
+            izz = read_float_child_element(xml, "izz", 1)
+            ixy = read_float_child_element(xml, "ixy", 0)
+            ixz = read_float_child_element(xml, "ixz", 0)
+            iyz = read_float_child_element(xml, "iyz", 0)
+            pose = Eigen::Isometry3.new
+             if inertial_elements = xml.elements["inertial"]
```

Comment:
What do you mean? 
I need to check the presence of inertial for get the inertial's pose (the cog). Otherwise it would pick the the link's pose.
For the above cases it was not necessary to do that (they passed the test). I think because there is no other place the mass or the inertia is set.